system-int-test/test/cmr/system_int_test/search/collection_directory_pages_test.clj

```diff
@@ -0,0 +1,240 @@
+(ns cmr.system-int-test.search.collection-directory-pages-test
+  "This tests a running CMR site's directory links at all levels: top-most,
+  eosdis, and provider."
+  (:require [clj-http.client :as client]
+            [clojure.test :refer :all]
+            [clojure.string :as str]
+            [cmr.search.site.routes :as r]
+            [cmr.system-int-test.data2.core :as d]
+            [cmr.system-int-test.utils.index-util :as index]
+            [cmr.system-int-test.utils.ingest-util :as ingest]
+            [cmr.system-int-test.utils.tag-util :as tags]
+            [cmr.mock-echo.client.echo-util :as e]
+            [cmr.system-int-test.system :as s]
+            [cmr.transmit.config :as transmit-config]
+            [cmr.umm-spec.models.umm-common-models :as cm]
+            [cmr.umm-spec.test.expected-conversion :as exp-conv]))
+
+(def ^:private base-url (format "%s://%s:%s"
```

Comment:
I think the base-url should end with a "/" to match what is being used in the source code though I'm guessing all of the tests that check this base-url are probably adding a "/" when used.